### ISSUE TITLE
Replying via "Reply" button causes status update lag

### ISSUE THREAD

**Issue by Dil3mm4** [telegramdesktop/tdesktop#4408]
#### Steps to reproduce
1. Telegram notification service on
2. Reply via Telegram Notification "Reply" Button
3. You'll result online for a lot of time even if the window is not focused

### Expected behaviour
Send reply and update user status in real-time (that should be set based on message sent timestamp)

### Actual behaviour
Reply gets sent but the status update it's laggy (not matching the message sent timestamp).
Basically it states that you're online even if you're not.  

### Configuration
**Operating system:**
Windows 10

**Version of Telegram Desktop:**
1.2.6

**Used theme**:
Default

**Comment by stek29** [comment]
iirc online timestamp is added by server when you send a message.
Probably TDesktop has to explicitly reset it after sending the message.

**Comment by tdesktop-closer** [comment]
Hey there!

We're automatically closing this issue since there was no activity in this issue since 405 days ago. We therefore assume that the user has lost interest or resolved the problem on their own. Closed issues that remain inactive for a long period may get automatically locked.

Don't worry though; if this is in error, let us know with a comment and we'll be happy to reopen the issue.

Thanks!

<sub>(Please note that this is an [automated](https://github.com/auchri/github_auto_closer) comment.)</sub>
